code/atom/UpdateOverlays.dm

```diff
@@ -271,15 +268,12 @@ ClearSpecificOverlays(1, "key0", "key1", "key2") 	//Same as above but retains ca
 
 /atom/proc/GetOverlayImage(var/key)
 	RETURN_TYPE(/image)
-	if (!src.overlay_refs)
-		src.overlay_refs = list()
+	if (!length(src.overlay_refs))
+		return
```

Comment:
is this correct

Reply:
yeh, if theres nothing we can return early